Extra info examples, connect info and define pages (#485)
surrealdb/docs.surrealdb.com: doc-surrealdb_versioned_docs/version-1.x/surrealql/statements/define/index.mdx

# DEFINE statement

> The DEFINE statement can be used to specify items including authentication access and behaviour, global parameters, table configurations, table events, schema definitions, and indexes.

Reviewer: I am unsure if 'items' is a good description here.  Maybe 
```suggestion
The DEFINE statement can be used to specify  instructions to the schema such as  authentication access and behaviour, global parameters, table configurations, table events, analyzers, and indexes.
```

	| INDEX [ IF NOT EXISTS ] @name ON [ TABLE ] @table [ FIELDS | COLUMNS ] @fields
		[ UNIQUE | SEARCH ANALYZER @analyzer [ BM25 [(@k1, @b)] ] [ HIGHLIGHTS ] ]
]
```

> The counterpart to `DEFINE` is the [INFO](/docs/surrealdb/surrealql/statements/info) command, which can be used to see which definition statements currently exist.

Reviewer: ```suggestion
The [INFO](/docs/surrealdb/surrealql/statements/info) statement can be used to see which definition statements currently exist in a database connection.
```
